.eslintrc.js

```diff
@@ -2,6 +2,9 @@ module.exports = {
   root: true,
   extends: ["universe/native"],
   plugins: ["react-hooks"],
+  settings: {
+    "import/ignore": ["react-native"],
+  },
```

Comment:
If you added this to fix an error, it may concerns your environment (I have the same on Windows). You must remove that before commit.
Please, remove that.